packages/iputils/iputils.spec

```diff
@@ -1,10 +1,10 @@
 Name: %{_cross_os}iputils
-Version: 20200821
+Version: 20210202
 Release: 1%{?dist}
 Summary: A set of network monitoring tools
 License: GPL-2.0-or-later AND BSD-3-Clause
 URL: https://github.com/iputils/iputils
-Source0: https://github.com/iputils/iputils/archive/s%{version}.tar.gz#/iputils-s%{version}.tar.gz
+Source0: https://github.com/iputils/iputils/archive/%{version}.tar.gz#/iputils-%{version}.tar.gz
```

Comment:
This is super weird. https://github.com/iputils/iputils/issues/269

Reply:
Yeah, hopefully they don't change it back :+1: